public/js/main.js

```diff
@@ -129,7 +129,7 @@ $(function () {
        { data: "email" },
        { data: "id",
          render: function (val) {
-           return '<a href="/admin/user/'+val+'" target="_blank">Edit</a>'
+           return '<a href="./user/' + val + '" target="_blank">Edit</a>'
```

Comment:
Shouldn't this be an absolute url too?

Reply:
This js code is loaded client side in the browser, where the APP_URL env var is not known (in the current situation). So I think this is the easiest solution.